src/sparseml/pytorch/utils/module.py

```diff
@@ -975,6 +976,7 @@ def _runner_batch(
 
             # loss calculation
             losses = self._loss(data, pred)
+            losses[DEFAULT_LOSS_KEY] /= self._num_accumulated_batches
```

Comment:
is this a bugfix since it didn't divide before?

Reply:
Yes. Gradient accumulation was in essence a ghost feature before, as it wasn't utilized or exposed to the user. But this does fix the logic. Also since your comment I've made an update to reset the loss to the un-normalized value after gradient calculations. This only impacts the value of the loss logged and makes sure it's printed correctly